src/components/chips/js/chipsDirective.js

```diff
@@ -136,49 +142,99 @@
       }
     };
     function compile(element, attr) {
-      var userTemplate = attr['$mdUserTemplate'];
-      var chipEl = userTemplate.find('md-chip');
-      var chipHtml;
-      if (chipEl.length === 0) {
+      function getWorkerChip(el) {
```

Comment:
vestige of a layout when the input control was inside a chip. renamed to input container